testinfra/common/test_tor_mirror.py

```diff
@@ -0,0 +1,58 @@
+import pytest
+
+
+def test_tor_mirror_present(File):
+    """
+    Ensure the FPF mirror of the Tor apt repo, tor-apt.freedom.press,
+    is configured. This repository required manual updating with current
+    tor releases, to avoid breakage of untested updates.
+    """
+    f = File('/etc/apt/sources.list.d/tor_apt_freedom_press.list')
+
+    regex = ('^deb http:\/\/tor-apt\.freedom\.press trusty main$')
+    assert f.contains(regex)
+
+
+def test_tor_mirror_fingerprint(Command):
+    """
+    Ensure the FPF tor mirror repo has the correct fingerprint on the
+    associated signing pubkey. We don't use the SecureDrop Release Signing
+    Key for the Tor mirror: packages are identical to the official Tor repo,
+    so signatures match, as well.
+    """
+    c = Command('apt-key finger')
+    tor_gpg_pub_key_info = """/etc/apt/trusted.gpg.d/deb.torproject.org-keyring.gpg
+-----------------------------------------------------
+pub   2048R/886DDD89 2009-09-04 [expires: 2020-08-29]
+      Key fingerprint = A3C4 F0F9 79CA A22C DBA8  F512 EE8C BC9E 886D DD89
+uid                  deb.torproject.org archive signing key
+sub   2048R/219EC810 2009-09-04 [expires: 2018-08-30]"""
+
+    assert c.rc == 0
+    assert tor_gpg_pub_key_info in c.stdout
```

Comment:
@conorsch and I just had a brief discussion about this: this test is clearly passing _but_ we had discussed a workflow where we resign Tor packages on our mirror with the SecureDrop release key, so this key should be gone. If we keep this as is, we'll need to act prior to the `Valid-Until` expires (as in, do QA and update the packages on the mirror before the package expires).